database/init_db.go

```diff
@@ -0,0 +1,67 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/lib/pq"
+	"os"
+)
+
+const (
+	host     = "localhost"
+	port     = 5432
+	user     = "" // the db_user from postgres-cm.yaml, ie: "admin", "ps_user".
+	password = "" // the db_password from postgres-cm.yaml
+	dbname   = "" // the db_name from postgres-cm.yaml, ie: "postgres", "test_db".
+)
+
+func main() {
+	// connect to the DB.
+	psqlInfo := fmt.Sprintf("host=%s port=%d user=%s "+
+		"password=%s dbname=%s sslmode=disable",
+		host, port, user, password, dbname)
+
+	// postgres is the driver type.
+	db, err := sql.Open("postgres", psqlInfo)
+	if err != nil {
+		panic(err)
+	}
+
+	defer db.Close()
+
+	// SQL instruction to create a table.
+	sqlStatement := `
+	CREATE TABLE public.test_objects (
+		"id" serial PRIMARY KEY,
+		"api_version" varchar NOT NULL,
+		"kind" varchar NOT NULL,
+		"name" varchar NOT NULL,
+		"namespace" varchar NOT NULL,
+		"resource_version" varchar NULL,
+		"data" jsonb NOT NULL
+	);
```

Comment:
Could we add here two more metadata fields? UPDATED_TS and CREATED_TS with timestamps for troubleshooting purposes? In my experience those are important fields to have as early as possible.

Reply:
@maruiz93, as you asking for fields that are already contained in the JSON?

Reply:
No, I'm asking for fields with the information of "when was this registry updated in the kubearchive database" (UPDATED_TS) and "when was this entry first inserted in the kubearchive database" (CREATED_TS). These are meant to be kubearchive metadata values for troubleshooting purposes for kubearchive. This data isn't meant to be retrieved through the kubearchive database.

Reply:
@maruiz93 I added the two fields. the created_ts has to be not null when we insert an object and updated_ts.
I also modify the testdata to have these values.

Reply:
In my experience, the usual approach is to set both updated and created timestamps when the entry is inserted in the database and from that point just update the updated ts if needed. I would prefer to follow this approach and set both fields as `NOT NULL`, but as the feature is covered, it's ok if you want to leave it as is.

For the data type part though, I would prefer to use a proper timestamp instead of a varchar (see [PostgreSQL datatypes for reference](https://www.postgresql.org/docs/current/datatype.html)). The advantages of doing so is that we don't need to deal with setting a standard format for the timestamp and checking, the database will ensure that the content of the field make sense for us.

Reply:
thanks!